error.js

```diff
@@ -1,75 +1,103 @@
+let factory;
 
-var error = {};
+class AssertionError extends Error {
+  constructor (message, actual, expected) {
+    if (factory == null) factory = require("./");
```

Comment:
so I switched it to a lazy require and everything works. kinda janky but it works.